torch/_inductor/sizevars.py

```diff
@@ -36,6 +36,29 @@
 log = logging.getLogger(__name__)
 
 
+def evaluate_expr(
+    shape_env: ShapeEnv,
+    expr: Union[sympy.Basic, bool],
+    axioms: Optional[Tuple[sympy.Expr]] = None,
+    var_to_range: Optional[Tuple[Tuple[sympy.Symbol, ValueRanges[Any]]]] = None,
+) -> bool:
+    if expr in (True, False):
+        return bool(expr)
+
+    try:
+        simplified = shape_env._maybe_evaluate_static(
+            expr,
+            axioms=axioms,
+            var_to_range=var_to_range,
+        )
+        if simplified is not None:
+            return bool(simplified)
+    except Exception:
+        log.debug("Could not simplify  %s", expr)
```

Comment:
```suggestion
        log.debug("Could not simplify  %s", expr, exc_info=True)
```
Include exception info too

Reply:
Thanks for the suggestion. Updated the code as suggested.